src/test/chain-simulator/miniblocks.cs-e2e.ts

```diff
@@ -0,0 +1,123 @@
+import axios from 'axios';
+
+const CHAIN_SIMULATOR_URL = 'http://localhost:8085';
+const API_SERVICE_URL = 'http://localhost:3001';
+
+describe('Miniblocks e2e tests with chain simulator', () => {
+  beforeAll(async () => {
+    try {
```

Comment:
could have used the common chain simulator initializer function

Reply:
done